aspnet/testing/unit-testing/sample/test/PrimeWeb.UnitTests/project.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "version": "1.0.0-*",
+  "description": "PrimeWeb.UnitTests Class Library",
+  "authors": [ "stevesmith (@ardalis)" ],
+  "tags": [ "" ],
+  "projectUrl": "",
+  "licenseUrl": "",
+  "frameworks": {
+    "dnx451": { },
+    "dnxcore50": {
+      "dependencies": {
+        "Microsoft.CSharp": "4.0.1-beta-23409",
+        "System.Collections": "4.0.11-beta-23409",
+        "System.Linq": "4.0.1-beta-23409",
+        "System.Runtime": "4.0.21-beta-23409",
+        "System.Threading": "4.0.11-beta-23409"
+      }
+    }
+  },
+  "dependencies": {
+    "PrimeWeb": "1.0.0-*",
+    "xunit": "2.1.0-*",
```

Comment:
Samples targeting released versions (beta8) should use fixed package dependencies.